src/tests/topp/ConvertTSVToTraML_2_output.TraML

```diff
@@ -113,7 +111,6 @@
         <cvParam cvRef="MS" accession="MS:1000827" name="isolation window target m/z" value="501" unitCvRef="MS" unitAccession="MS:1000040" unitName="m/z"/>
       </Precursor>
       <Product>
-        <cvParam cvRef="MS" accession="MS:1000041" name="charge state" value="2"/>
```

Comment:
@grosenberger you should also change src/tests/topp/ConvertTSVToTraML_1_input.csv and it seems you changed the header name (why?) as before we had

- FragmentType
- FragmentCharge
- FragmentSeriesNumber

and now FragmentCharge is renamed to ProductCharge which is a bit inconsistent and loses the information in the TraML